fastlane/lib/fastlane/plugins/template/lib/fastlane/plugin/%plugin_name%/actions/%plugin_name%_action.rb.erb

```diff
@@ -6,11 +6,11 @@ module Fastlane
       end
 
       def self.description
-        %q{<%= summary %>}
+        '<%= summary %>'
```

Comment:
I approve this change, however we need some basic escaping, could add you add this? I'd also prefer to use `"` as it's confusing for non-Ruby developers on why we have 2 different types of string